packages/gatsby-source-mongodb/src/preprocess-aggregations.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Filters out empty or wrong formatted aggregations.
+ * Re-formats unnamed aggregations (shorthand format) into named format.
+ * ```
+ * {
+ *  'collection_1' : {
+ *    'aggregation_1': [ … ],
+ *    'aggregation_2': [ … ],
+ *  },
+ *  'collection_2' : {
+ *    'aggregation_1': [ … ],
```

Comment:
The linting errors are because of irregular whitespace characters in these lines:

```
   7:27  error    Irregular whitespace not allowed  no-irregular-whitespace
   8:27  error    Irregular whitespace not allowed  no-irregular-whitespace
  11:27  error    Irregular whitespace not allowed  no-irregular-whitespace
```

Reply:
Thanks! I committed a fix. I will never understand why my VSCode sometimes adds non-whitespace characters on its own, have this problem for years now.. Also activating "Whitespace Rendering" doesn't really fix this for me as it's often hard to stop if one of those tiny whitespace-bullets is missing. But thanks anyway!